backup-loop.sh

```diff
@@ -419,9 +438,18 @@ while true; do
     retry ${RCON_RETRIES} ${RCON_RETRY_INTERVAL} rcon-cli save-all flush
     retry ${RCON_RETRIES} ${RCON_RETRY_INTERVAL} sync
 
+    if [[ $PRE_BACKUP_SCRIPT_FILE ]]; then
+      "$PRE_BACKUP_SCRIPT_FILE"
+    fi
```

Comment:
Rather than write out the command contents to temp files, I'm thinking you could directly eval them, such as
```suggestion
    if [[ $PRE_BACKUP_SCRIPT ]]; then
      eval "$PRE_BACKUP_SCRIPT"
    elif [[ $PRE_BACKUP_SCRIPT_FILE ]]; then
      "$PRE_BACKUP_SCRIPT_FILE"
    fi
```

Reply:
Been a long time I know, but the reason I skipped out on `eval` was to not force `set -euo pipefail` on the provided scripts. While it's great for mc-backup it might not work well for custom pre/post-backup scripts.

Reply:
`bash -c` could have worked then, but the file approach makes sense since the user has option to provide file. 

My worry was about an abnormal failure leaving temp files around.  And the best way to solve a problem is to avoid the problem.